pfp/interp.py

```diff
@@ -552,8 +553,19 @@ def define_natives(cls):
 		if len(cls._natives) > 0:
 			return
 
-		glob_pattern = os.path.join(os.path.dirname(__file__), "native", "*.py")
-		for filename in glob.glob(glob_pattern):
+		pythonfiles = []
+
+		mydir = os.path.dirname(__file__)
+		if '.zip' in mydir:
```

Comment:
Shouldn't this be `if mydir.endswith('.zip')`?

Reply:
I think it's different than .endswith() because the zip file can be one among many directories in the path and is not necessarily the last one.